dashboard/app/controllers/browser_events_controller.rb

```diff
@@ -42,6 +44,37 @@ def put_logs
     render status: :internal_server_error, json: {error: exception}
   end
 
+  # POST /put_metric_data
+  def put_metric_data
+    body = JSON.parse(request.body.read)
+
+    return render status :bad_request, json: {message: 'missing required params: metricData'} unless body["metricData"]
+
+    metrics = body["metricData"].map {|datum| convert_metric_datum(datum)}
+    Cdo::Metrics.push(METRIC_NAMESPACE, metrics)
```

Comment:
where do you flush the data? Not sure if it will to the server if you dont call flush

Reply:
Oh do we need to flush the buffer manually? I thought it would automatically flush after its batch size had been reached? I tried manually reducing the batch size to 2 and I thought I saw metrics post but maybe I'm misunderstanding.

Reply:
Resolved this on Slack